### ISSUE TITLE
Attempt to set temperature high limit at 74F sets to 73.99

### ISSUE THREAD

**Issue by DG12** [ruuvi/com.ruuvi.station.webui#216]
Is this a F to C to F rounding issue?

BC
(74-32)*(5/9)     #. Degrees Fahrenheit  to Centigrade 
23.33333310

(.*(9/5))+32.    # and back
73.99999958

![image](https://user-images.githubusercontent.com/2365961/190465992-8583213b-f42e-49ad-84f9-3b8714afee1e.png)

Of course there are many other values that need "nice" rounding, like 76-> 75.99 ,78-> 78.01

**Comment by laurijamsa** [comment]
Do we have this same behaviour on Android and iOS as well?

**Comment by DG12** [comment]
Android does not have a input box to enter a value however,
lower ----upper limit slider will not set to 74 (to 76 )even in landscape.

**Comment by laurijamsa** [comment]
I've reproduced this. It happens also when you slide the slider:

https://user-images.githubusercontent.com/1311912/197190909-f634bf2d-f9b6-4c4c-abda-5d770ec54510.mov

On Android, it doesn't show the F values like this. If I set 74F upper limit on Android, it shows 74F. But when I after this refresh the webUI, the webUI shows 73,99F. So, let's check the Android implementation and make the webUI work similar way as Android.
